### DISCUSSION TITLE
Disable Cross Device Experience Host

### DISCUSSION THREAD

**Discussion by ProgrammerBruce** [undergroundwires/privacy.sexy#464] [General]
Was it already considered to add a script to Disable or Remove Cross Device Experience Host?

Maybe it would fit in the "Remove Bloatware > Remove non-essential services" category, though it is related to "Remove Bloatware > Remove Windows Apps > Remove Phone Apps".

I do not yet know what issues arise if it is removed or disabled.

The package name may be MicrosoftWindows.CrossDevice.
```
Get-AppxPackage -Name *Device*
Get-AppxProvisionedPackage -Online | where {$_.PackageName -like '*Device*'}
```

From winget, maybe it is this:
```
winget search 9NTXGKQ8P7N0
Name                         Id           Version Source
----------------------------------------------------------
Cross Device Experience Host 9NTXGKQ8P7N0 Unknown msstore
```

To disable it, a registry setting might be:
```
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Mobility]
"CrossDeviceEnabled"=dword:00000000
```

In Windows 11 24H2 build 26100.2454 GUI, the component is listed in "Settings > System > System components".  I do not see it listed in build 26100.2314.

**Comment by undergroundwires** [comment]
I've done some additional research and [added the script](https://github.com/undergroundwires/privacy.sexy/commit/20944751bcaedcd475a69a99b774982c6efb89d1), it will be released in next patch. Thank you @ProgrammerBruce. It's good to create issues rather than discussions for these for better managing TODO tasks.

**Comment by villarrealanthony056-ops** [comment]
Get-AppxPackage -Name *Device*
Get-AppxProvisionedPackage -Online | where {$_.PackageName -like '*Device*'}

**Comment by Midnightmadwalk-coder** [comment]
@villarrealanthony056-ops how can i  find the option to remove crossdeviceresume.exe? i cant find it in the app in search results. Thanks in advance.
